src/main/java/org/openrewrite/kotlin/tree/K.java

```diff
@@ -320,6 +320,66 @@ public <S> S service(Class<S> service) {
         }
     }
 
+    /**
+     * In Kotlin all expressions can be annotated with annotations with the corresponding annotation target.
+     */
+    @Getter
+    @SuppressWarnings("unchecked")
+    @FieldDefaults(makeFinal = true, level = AccessLevel.PRIVATE)
+    @EqualsAndHashCode(callSuper = false, onlyExplicitlyIncluded = true)
+    final class AnnotatedExpression implements K, Expression {
+
+        @With
+        UUID id;
+
+        @With
+        Space prefix;
```

Comment:
I suppose the prefix could instead be pushed down to the first annotation, if we don't want it here.

Reply:
Yeah, I think the prefix should be on the annotation to prevent as many unintentional changes from J as possible.

`ExpressionStatement`, `StatementExpression`, etc. work in a similar way. The `K` tree is just a pass-through to J.

Reply:
OK. Thanks for the feedback. I will move that prefix then.